### ISSUE TITLE
how to backup my sql files

### ISSUE THREAD

**Issue by kpopdev** [SideProjectGuys/invite-manager-bot#184]
hi sorry if im noob but how can i backup my sql files if i wanted to move to my new vps or pc.
or where the sql files and my database saves.

**Comment by ghost** [comment]
sudo mysqldump -u root -p im_0 > PATH_TO_BACKUP_OF_IM0.sql
sudo mysqldump -u root -p im_1 > PATH_TO_BACKUP_OF_IM1.sql

If you used a user other than root while creating the databases, use that instead. The root user doesn't work without sudo.
Replace PATH_TO_BACKUP with where you want to save the backups. For example: /home/username/Desktop/

You can later restore those the same way you initialized the databases when setting up the bot. Just use your backup files instead of the .sql files provided in the scripts directory.

**Comment by kpopdev** [comment]
and how to restore it?when i have new vps @BreezeQS

**Comment by chaun14** [comment]
BreezeQS answered you in his message above, 
`You can later restore those the same way you initialized the databases when setting up the bot. Just use your backup files instead of the .sql files provided in the scripts directory.`

**Comment by ghost** [comment]
Yeah. And just in case you break something and need to roll back to a backup, rather than moving to a different place; you can drop the broken databases, create the new ones and import your backup .sql files.
